docs/sources/write/shortcodes/index.md

```diff
@@ -296,3 +289,17 @@ This Markdown renders as:
 ```markdown
 {{</* myshortcode */>}}
 ```
+
+## About version inference
+
+Version inference enables the use of absolute paths that resolve correctly, irrespective of version.
+It uses special syntax using angle bracket delimiters like `<GRAFANA VERSION>`.
+
+As a convention, use the name of the target project, with spaces but no hyphens or underscores, all upper-case.
+For example, `grafana` becomes `GRAFANA`, `grafana-cloud` becomes GRAFANA CLOUD.
+
+The special syntax `<SOMETHING VERSION>`, is inferred from the version of the page.
```

Comment:
I don't understand this part. The person writing the shortcode has to enter all of this—what exactly is being inferred?

Reply:
I can see why this is confusing. The value of the substitution is inferred from the version of the page.

Perhaps the following is more clear:

```suggestion
The special syntax `<SOMETHING VERSION>` is substituted by the version that is inferred from the page's URL.
```

Reply:
Yeah that makes more sense to me